src/components/quotes/quotePage.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react';
+import classes from '../quote.module.css';
+
+const Quote = () => {
+  const [quote, setQuote] = useState('');
+  const [author, setAuthor] = useState('');
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+  const [mounted, setMounted] = useState(false);
+
+  useEffect(() => {
+    setMounted(true);
+    return () => {
+      setMounted(false);
+    };
+  }, []);
+
+  useEffect(() => {
+    const fetchQuote = async () => {
+      setLoading(true);
+      setError(false);
+
+      try {
+        const res = await fetch('https://api.api-ninjas.com/v1/quotes?category=happiness', {
+          headers: {
+            'X-Api-Key': 'jzzfew774Kqg+nf2AkhCAw==pIwj1VNrBEBwQ0lW',
+            'Content-Type': 'application/json',
+          },
+        });
+
+        const data = await res.json();
+        if (mounted) {
+          setQuote(data[0].quote);
+          setAuthor(data[0].author);
+          setLoading(false);
+        }
+      } catch (error) {
+        setError(true);
+        setLoading(false);
+      }
+    };
+    fetchQuote();
+  }, [mounted]);
+
+  return (
+    <div className={classes.content}>
+      {loading && <div>Loading...</div>}
+      {error && <div>Something went wrong...</div>}
+      {loading && !error && (
+        <div>
+          <p>
+            &#34;
+            {quote}
+            &#34;
+            {author}
+          </p>
```

Comment:
- [ ] In order to match the given wireframe, it would be better if the quotes page shows quotes as seen on the images of the wireframe, but currently, the quotes are displayed on the calculator page, and the quotes page remains blank.

![quotes](https://user-images.githubusercontent.com/53124548/230613249-b772a143-5364-485f-87f3-e13b81a81da3.png)